memgpt/schemas/memgpt_message.py

```diff
@@ -12,7 +13,9 @@ class BaseMemGPTMessage(BaseModel):
 
     @field_serializer("date")
     def serialize_datetime(self, dt: datetime, _info):
-        return dt.now(timezone.utc).isoformat()
```

Comment:
@4shub @goetzrobin FYI this was a pretty bad bug that was previously causing all message streaming response chunks to have newly created timestamps